buildspec.yml

```diff
@@ -33,25 +33,22 @@ phases:
       - tox -e flake8,twine
 
       # run unit tests
-      - tox -e py36,py37 test/unit
+      - tox -e py38,py39,py310 test/unit
 
       # define tags
-      - GENERIC_TAG="$FRAMEWORK_VERSION-pytorch-$BUILD_ID"
       - DLC_CPU_TAG="$FRAMEWORK_VERSION-dlc-cpu-$BUILD_ID"
       - DLC_GPU_TAG="$FRAMEWORK_VERSION-dlc-gpu-$BUILD_ID"
       - DLC_EIA_TAG="$EIA_FRAMEWORK_VERSION-dlc-eia-$BUILD_ID"
 
       # run local CPU integration tests (build and push the image to ECR repo)
-      - test_cmd="IGNORE_COVERAGE=- tox -e py36 -- test/integration/local -vv -rA -s --build-image --push-image --dockerfile-type pytorch --region $AWS_DEFAULT_REGION --docker-base-name $ECR_REPO --aws-id $ACCOUNT --framework-version $FRAMEWORK_VERSION --processor cpu --tag $GENERIC_TAG"
```

Comment:
Why do we have 2 tests here in the first place?

Reply:
I think the original idea behind having these tests was to create a custom image similar to our DLCs. However, it seems redundant given we have both the DLC CPU and GPU images available for the testing. Furthermore, we run the integration tests for two DLC versions i.e. `2.0.0` and `2.0.1` (these are the versions currently supported by DLC team). Once DLC team releases the DLC containers for later PyTorch versions, we will include those versions in the integration tests as well, while taking care to remove the testing for versions which will not be supported by the DLC team at that time.